src/actions/ably-actions.js

```
@@ -1,31 +1,39 @@
+
+const getAblyClient = async () => {
+  if (!AblyApiKey) return null;
+  if (realtimeAbly) return realtimeAbly;
+  const Ably = (await import("ably")).default;
+  realtimeAbly = new Ably.Realtime(AblyApiKey);
+  return realtimeAbly;
+};
 
-export const subscribeToAblyChannel = (channel, callback) => {
-  if (!realtimeAbly) {
+export const subscribeToAblyChannel = (channel, callback) => async (dispatch) => {
```

ESLint

[error] 'dispatch' is defined but never used.

(no-unused-vars)